### DISCUSSION TITLE
Is there a way to add text editor like CK Editor in Orchadcore CMS

### DISCUSSION THREAD

**Discussion by syedshadab19** [OrchardCMS/OrchardCore#11875] [Q&A]
HI i wanted to know if there is a way to ck editor to the orchardcore or any other editor?

**Comment by MikeAlhayek** [comment]
Yes there is. https://docs.orchardcore.net/en/latest/docs/reference/modules/Markdown/#html-editor where is already a Wysiwyg editor that you can use. but you are more than welcome to create your own.

**Comment by Skrypt** [comment]
Yes, it is possible. But it cannot be distributed as part of `Orchard Core` source code because of licensing terms.

**Reply by hishamco** [reply]
@Skrypt I remebered I used CKEditor in many websites in the past, I'm not sure if some versions of CKEditor are permitted to distribute

**Comment by agriffard** [comment]
https://github.com/ThisNetWorks/ThisNetWorks.OrchardCore.CKEditor

**Reply by syedshadab19** [reply]
Hi @agriffard  is there a way to make the CK editor to have full-featured t? having all the functionalities currently, it has only the basic features in toolbar.
